### ISSUE TITLE
Allow downloads of selected information from the interface

### ISSUE THREAD

**Issue by Emma-Manners** [chembl/GLaDOS#1251]
Re Helpdesk ticket 407833, 376569 (and others)

It's not currently possible to download a CSV/TSV table with selected columns or compounds from the interface, several users have asked if this is possible and it would be a useful feature.

**Comment by nclopezo** [comment]
I have noticed this request as well. However, I have always given it a low priority because this can be done quite easily with applications like excel. And this feature can add some complexity to the downloads. I can add it in the future though.

**Comment by Emma-Manners** [comment]
Yes, you're right and I agree that it is low priority. If this is elsewhere then please feel free to close the ticket. Thanks!
